src/components/Footer.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { translate } from 'react-i18next';
```

Comment:
I was having a lot of trouble figuring out where this was declared, because the react-i18next docs make no mention of it. Turns out `gatsby-plugin-i18next` is using the _ancient_ v7.x when they are on v10.x now.

Only reason I was digging is because the unit tests were borked. Still untangling that mess locally.